lib/standard/merges_settings.rb

```diff
@@ -29,6 +29,9 @@ def parse_standard_argv(argv)
         if arg == "--fix"
           cli_flags[:auto_correct] = true
           cli_flags[:safe_auto_correct] = true
+        elsif arg == "--no-fix"
+          cli_flags[:auto_correct] = false
+          cli_flags[:safe_auto_correct] = false
```

Comment:
What happens if both are specified? I think we should have a test that last-in-wins

Reply:
Done in d16fd43f72c0a1d7bad83a036354c03c7d90d7a6